### ISSUE TITLE
New Home Page Layout

### ISSUE THREAD

**Issue by philipbaileynar** [namc-utah/namc-usu.org#3]
Once we get the new MAPIT application up and running, I am picturing two boxes on the web site home page that provide easy access to the places we anticipate visitors to the site wanting to go. What follows is a crude mockup that someone with better graphical design skills than me can implement:

<img width="1075" alt="Screen Shot 2022-01-21 at 3 52 19 PM" src="https://user-images.githubusercontent.com/8645379/150614578-5f9d4877-5c3d-4aad-90ac-2eafebf1fd83.png">

FYI @Feldspar5 @tybot @andrew-caudillo

**Comment by Feldspar5** [comment]
I agree but think we could go for even cleaner: three large boxes - the above two and a 3rd for general NAMC info (staff, recent research etc.  Maybe just the boxes with our mission statement.
